README.md

```diff
@@ -108,6 +107,28 @@ ansible-playbook -i ./BraveButtonsConfig/ansible/<inventory file name> \
                  --ask-vault-pass
 ```
 
+# How to release a new version & update the entire Button system (server and RPi fleet)
+
+1. decide on an appropriate version number for the new version
+
+1. update CHANGELOG.md by moving everything in `Unreleased` to a section for the new version
+
+1. make a new git tag in both the `BraveButtons` and `BraveButtonsConfig` repositories with the version number
+
+1. update the `production` branch, ensuring that it is in sync with the tag for the new version
```

Comment:
Could you please include the git command for this too to make sure that we're always doing it the same way and don't end up with any icky merge commits in weird places?